Bootloaders/DFU/BootloaderDFU.c

```diff
@@ -135,19 +135,27 @@ void Application_Jump_Check(void)
 		/* Check if the device's BOOTRST fuse is set */
 		if (!(BootloaderAPI_ReadFuse(GET_HIGH_FUSE_BITS) & ~FUSE_BOOTRST))
 		{
+		#if __GNUC__ == 8 && __GNUC_MINOR__ == 3
```

Comment:
probably would be okay to check to see if `__GNUC__` is > than 6 here, and not for a specific version. 

I can test that on my Windows machine too, as I have it set up to quickly switch between a bunch of different versions of avr-gcc.

Reply:
That'd be nice. I pinned it to 8.3 as it's the only one I've gotten working yet. I had other issues with 7.2